test/test_moves_files_to_local_machine.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+# You can run it from any directory.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
@artem-zinnatullin, what is we `cd` to `test` folder before looping through each test and on test end `pop` original folder?